### PR TITLE
OCPBUGS16018: CMA cannot be used with multi namespaces but the doc says we can

### PR DESCRIPTION
**Pull request by mburke5678**
https://issues.redhat.com/browse/OCPBUGS-16018

### CODE REVIEW

**Comment by mburke5678** [comment]
@jkyros Is this an issue you can review? Thanks in advance!

**Comment by jkyros** [comment]
yep, this is correct
/lgtm

File: modules/nodes-cma-autoscaling-custom-install.adoc [hunk 1]
```diff
@@ -112,7 +112,7 @@ spec:
         maxSize: "50"
   serviceAccount: {}
 ----
-<1> Specifies the namespaces that the custom autoscaler should watch. Enter names in a comma-separated list. Omit or set empty to watch all namespaces. The default is empty.
+<1> Specifies a single namespace where the custom autoscaler is to scale applications. Omit or set empty to scale applications in all namespaces. This field should have one namespace or be empty. The default is empty.
```
Comment (agantony) [inline modules/nodes-cma-autoscaling-custom-install.adoc:115]:
```suggestion
<1> Specifies a single namespace in which the custom autoscaler should scale applications. Leave it blank or leave it empty to scale applications in all namespaces. This field should have a namespace or be empty. The default value is empty.
```

File: modules/sd-nodes-cma-autoscaling-custom-install.adoc [hunk 2]
```diff
@@ -126,7 +126,7 @@ spec:
         maxSize: "50"
   serviceAccount: {}
 ----
-<1> Specifies a single namespace where the custom autoscaler is to scale applications. Omit or set empty to scale applications in all namespaces. The default is empty.
+<1> Specifies a single namespace where the custom autoscaler is to scale applications. Omit or set empty to scale applications in all namespaces. This field should have one namespace or be empty. The default is empty.
```
Comment (agantony) [inline modules/sd-nodes-cma-autoscaling-custom-install.adoc:129]:
```suggestion
<1> Specifies a single namespace in which the custom autoscaler should scale applications. Leave it blank or leave it empty to scale applications in all namespaces. This field should have a namespace or be empty. The default value is empty.
```

**Review by agantony — COMMENTED** [review]
Added a few suggestions for your consideration; overall lgtm!

File: modules/sd-nodes-cma-autoscaling-custom-install.adoc (hunk 2 above)
Comment (mburke5678) [reply modules/sd-nodes-cma-autoscaling-custom-install.adoc]: @agantony Thanks for your comments. It does read better.

**Pull request merged**